configure.ac

```diff
@@ -1,25 +1,23 @@
 dnl configure.in for netatalk
 
-AC_INIT([netatalk], [m4_esyscmd([echo -n `cat VERSION`])])
+AC_INIT([netatalk],[m4_esyscmd([echo -n `cat VERSION`])])
```

Comment:
I think we can simplify this to `AC_INIT([netatalk],m4_normalize(m4_include([VERSION])))`

For the record: `echo -n` isn't portable